### PR TITLE
add abstract page content skeleton

### PR DESCRIPTION
**Pull request by MaHaWo**
- adds the visual element for the basic page described in #2 . 
- allows for passing in components to render them into the basic page
- has no functionality at the moment for buttons or links.
- created a dummy frontpage to have the `AbstractContent` component do something.

### CODE REVIEW

**Comment by MaHaWo** [comment]
One big question remains: where to put stylings? It seems that the idea is to apply them directly to wherever you need them in your component, but that seems to be an oddly non-reusable paradigm, since component styling and component functions do not necessarily correlate 100%? And having a global style sheet in which to maintain everything seems like a recipe for desaster?

**Comment by MaHaWo** [comment]
what tests would be necessary at this point? there is no functionality yet, so I didn't really write any tests either....

File: src/lib/components/Frontpage.svelte [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<script>
+	import { Button, Card, Gallery } from 'flowbite-svelte';
+	import { ArrowRightOutline } from 'flowbite-svelte-icons';
+
+	// Define the styles for the header, paragraph, arrow, and button
+	const headerStyle = 'mb-2 text-2xl font-bold tracking-tight text-gray-900 dark:text-white';
+	const paragraphStyle = 'mb-3 font-normal text-gray-700 dark:text-gray-400 leading-tight';
+	const arrowStyle = 'ms-2 h-6 w-6 text-white';
+	const buttonStyle = 'w-fit';
+	const galleryStyle = 'grid-cols-2 gap-y-12';
```
Comment (lkeegan) [inline src/lib/components/Frontpage.svelte:10]: 2 cols looks cramped on a small screen, maybe have 1 column by default, with 2 cols for medium+ size displays?

`'grid-cols-1 md:grid-cols-2'`

**Review by lkeegan — COMMENTED** [review]
looks good! I think that in general `<script>` is not the place for styling - I'd suggest putting them directly in `class=...` and if that would result in duplication I'd suggest making a re-usable component with this styling.

**Comment by lkeegan** [comment]
> what tests would be necessary at this point? there is no functionality yet, so I didn't really write any tests either....

Agree - we can look at adding some cypress ui tests at some point, but I wouldn't start doing this until we have pretty much solidified the design.

File: src/lib/components/Frontpage.svelte [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<!-- This is a dummy landing page that is only there to check that the AbstractComponent works as it should-->
+<Gallery class={galleryStyle}>
+	<Card class={cardStyle}>
+		<h5 class={headerStyle}>Was ist Mondey?</h5>
+		<p class={paragraphStyle}>
+			<i>Mondey</i> ist ein wissentschaftlich geprüftes Programm zure Dokumentation der Entwicklung von
+			Kindern bis 6 Jahren.
+		</p>
+		<Button class={buttonStyle}>Mehr Info <ArrowRightOutline class={arrowStyle} /></Button>
+	</Card>
```
[the remark this answers is not in the document]
Comment (MaHaWo) [reply src/lib/components/Frontpage.svelte:23]: was thinking about this. but given that this is a dummy page and Mrs. Pauen wanted to design the page herself, I didn´t go to these lengths. Or do you think we should use the cards therein/somewhere else? In that case I would suggest having a separate issue?

[the remark this answers is not in the document]
Comment (MaHaWo) [reply src/lib/components/Frontpage.svelte:23]: @lkeegan

**Pull request merged**